docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    }
+    ```
+
+5. To finish the negotiation, use `webrtc_add_ice_candidate()`, `webrtc_set_local_description()` or `webrtc_set_remote_description()` function:
+
+    ```c
+    /* After receiving all of ICE candidates from remote peer */
+    ret = webrtc_add_ice_candidate(webrtc, candidate);
+    ...
+    /* In case of an answerer */
+    ret = webrtc_set_local_description(webrtc, answer_desc);
+    ... or ...
+    /* In case of an offerer */
+    ret = webrtc_set_remote_description(webrtc, answer_desc);
+    ...
+    /* If the connection is established successfully, you'll get notified of WEBRTC_STATE_PLAYING by _state_changed_cb() */
+    ```
+6. To get notified of various negotiation states, set callbacks by using `webrtc_set_peer_connection_state_change_cb()`, `webrtc_set_signaling_state_change_cb()`, `webrtc_set_ice_gathering_state_change_cb()` and `webrtc_set_ice_connection_state_change_cb()` function:
+
+    ```c
+    void _peer_connection_state_change_cb(webrtc_h webrtc, webrtc_peer_connection_state_e state, void *user_data)
+    {
+        /* After setting both description and ICE candidates from remote peer, it'll be changed from WEBRTC_PEER_CONNECTION_STATE_CONNECTING to WEBRTC_PEER_CONNECTION_STATE_CONNECTED. */
+    }
+
+    void _signaling_state_change_cb(webrtc_h webrtc, webrtc_signaling_state_e state, void *user_data)
+    {
+        /* After setting local or remote description it'll be changed to WEBRTC_SIGNALING_STATE_HAVE_LOCAL_OFFER or WEBRTC_SIGNALING_STATE_HAVE_REMOTE_OFFER respectively. Both descriptions are set, it'll be changed to WEBRTC_SIGNALING_STATE_STABLE. */
+    }
+
+    void _ice_gathering_state_change_cb(webrtc_h webrtc, webrtc_ice_gathering_state_e state, void *user_data)
+    {
+        /* When setting local description, it'll be changed to WEBRTC_ICE_GATHERING_STATE_GATHERING. If the gathering work is done, it'll be changed to WEBRTC_ICE_GATHERING_STATE_COMPLETE. */
+    }
+
+    void _ice_connection_state_change_cb(webrtc_h webrtc, webrtc_ice_connection_state_e state, void *user_data)
+    {
+        /* After setting both description and ICE candidates from remote peer, it'll be changed from WEBRTC_ICE_CONNECTION_STATE_CHECKING to WEBRTC_ICE_CONNECTION_STATE_COMPLETED. */
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        ...
+        ret = webrtc_set_peer_connection_state_change_cb(webrtc, _peer_connection_state_change_cb, user_data);
+        ret = webrtc_set_signaling_state_change_cb(webrtc, _signaling_state_change_cb, user_data);
+        ret = webrtc_set_ice_gathering_state_change_cb(webrtc, _ice_gathering_state_change_cb, user_data);
+        ret = webrtc_set_ice_connection_state_change_cb(webrtc, _ice_connection_state_change_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+        /* Do negotiation */
+    }
+    ```
+
+<a name="media_render"></a>
+## Rendering Audio/Video Data
```

Comment:
For second headings, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Rendering audio/video data
```